command/operator_usage.go

```diff
@@ -245,6 +246,11 @@ func (c *OperatorUsageCommand) parseNamespaceCount(rawVal interface{}) (UsageRes
 	// don't error if the secret syncs key is missing
 	ret.secretSyncs, _ = jsonNumberOK(counts, "secret_syncs")
 
+	ret.acmeCount, ok = jsonNumberOK(counts, "acme_clients")
+	if !ok {
+		return ret, errors.New("missing acme_clients")
```

Comment:
I don't think we should error here - if the vault client version is greater than the vault server version, then `acme_clients` might be missing from the response

Reply:
Ah good point